bundler/lib/bundler/man/bundle-add.1.ronn

```diff
@@ -33,9 +33,15 @@ bundle add rails --group "development, test"
 * `--git`:
   Specify the git source for the added gem.
 
+* `--github`:
+  Specify the github source for the added gem.
+
 * `--branch`:
   Specify the git branch for the added gem.
 
+* `--ref`:
+  Specify the git ref for the added gem.
+
```

Comment:
Oh, I think you're going to also need to run `bin/rake man:build` from the `bundler` folder and commit the result to synchronize all documentation formats. Also, if you can rebase and squash into one commit, I think we'd be good to go unless CI says the opposite.

Reply:
Yep, everything's done!